xwiki-platform-core/xwiki-platform-rest/xwiki-platform-rest-model/src/main/java/org/xwiki/rest/model/jaxb/PropertyValue.java

```diff
@@ -100,4 +100,44 @@ public void setMetaData(java.util.Map<String, java.lang.Object> value)
     {
         this.metaData = value;
     }
+
+    @Override
+    public boolean equals(java.lang.Object o)
+    {
+        if (o instanceof PropertyValue) {
+            PropertyValue other = (PropertyValue) o;
+
+            boolean equals;
+            if (this.getMetaData() != null) {
```

Comment:
Why not use ``EqualsBuilder``?

Reply:
apparently the commons-lang dependency is not available in this module. I can add it, wasn't sure what the best for this case.